### ISSUE TITLE
Could not open disk image: Permission denied

### ISSUE THREAD

**Issue by manveru** [jedi4ever/veewee#737]
Full debug log is at https://gist.github.com/manveru/810b83da37478055bd98

I tried `chmod 0777`, `chgown manveru:kvm`, but every time i run `veewee build`, it ends up with:

```
xi github/jedi4ever/veewee % ls -la iso/ubuntu-13.04-server-amd64.iso
-rw-r--r-- 1 nobody kvm 735051776 Jun 24 10:08 iso/ubuntu-13.04-server-amd64.iso
```

So I tried locking permissions to `manveru:kvm` with `chattr +i`, but even then I get the same error.
I'm out of ideas why this is happening apart from digging through massive strace logs, any help would be appreciated.

**Comment by lacksfish** [comment]
For anybody still having this issue, throw the ISO file into `/var/lib/libvirt/images` and open from there.

**Comment by loxK** [comment]
@lacksfish I need it in another folder because of disk space

**Comment by zhouwg** [comment]
@billtang solution works fine for me in CentOS7,Thanks too much:

weiguo@weiguo:~$ ls -Z /home/weiguo/xp.iso 
-rwxrwxrwx. root root system_u:object_r:virt_content_t:s0 /home/weiguo/xp.iso
weiguo@weiguo:~$ sudo -u qemu file /home/weiguo/xp.iso 
[sudo] password for weiguo: 
/home/weiguo/xp.iso: cannot open (Permission denied)

weiguo@weiguo:~$ sudo chmod ugo+rx /home/weiguo/xp.iso 
weiguo@weiguo:~$ sudo -u qemu file /home/weiguo/xp.iso 
/home/weiguo/xp.iso: cannot open (Permission denied)

weiguo@weiguo:~$ sudo chmod ugo+rx /home/weiguo
weiguo@weiguo:~$ sudo -u qemu file /home/weiguo/xp.iso 
/home/weiguo/xp.iso: # ISO 9660 CD-ROM filesystem data 'GRTMPVOL_CN' (bootable)
weiguo@weiguo:~$ 


---------------------------------------------------------------------------
Unable to complete install: 'internal error: qemu unexpectedly closed the monitor: 2019-04-14 09:07:27.349+0000: Domain id=2 is tainted: high-privileges
2019-04-14T09:07:27.508170Z qemu-kvm: -drive file=/home/weiguo/xp.iso,format=raw,if=none,id=drive-ide0-0-1,readonly=on: could not open disk image /home/weiguo/xp.iso: Could not open '/home/weiguo/xp.iso': Permission denied'

Traceback (most recent call last):
  File "/usr/share/virt-manager/virtManager/asyncjob.py", line 89, in cb_wrapper
    callback(asyncjob, *args, **kwargs)
  File "/usr/share/virt-manager/virtManager/create.py", line 2553, in _do_async_install
    guest.start_install(meter=meter)
  File "/usr/share/virt-manager/virtinst/guest.py", line 498, in start_install
    doboot, transient)
  File "/usr/share/virt-manager/virtinst/guest.py", line 434, in _create_guest
    domain = self.conn.createXML(install_xml or final_xml, 0)
  File "/usr/lib64/python2.7/site-packages/libvirt.py", line 3725, in createXML
    if ret is None:raise libvirtError('virDomainCreateXML() failed', conn=self)
libvirtError: internal error: qemu unexpectedly closed the monitor: 2019-04-14 09:07:27.349+0000: Domain id=2 is tainted: high-privileges
2019-04-14T09:07:27.508170Z qemu-kvm: -drive file=/home/weiguo/xp.iso,format=raw,if=none,id=drive-ide0-0-1,readonly=on: could not open disk image /home/weiguo/xp.iso: Could not open '/home/weiguo/xp.iso': Permission denied
